run.py

```diff
@@ -133,6 +133,8 @@ def config_validator():
             backup_files.sort()
             for old_file in backup_files[:-3]:
                 os.remove(app.config['BACKUP_DEFAULT_DEST'] + old_file)
+        else:
+            print ('There is nothing to backup to ' + app.config['BACKUP_DEFAULT_DEST'])
```

Comment:
* remove  the space in between print and the paranthesis.
* the line is tool long.


This is a way to do it.

```python
            print('There is nothing to backup to ' 
                  + app.config['BACKUP_DEFAULT_DEST'])
```

The commit message indeed needs a hashtag for the issue number.

Note: There's an opportunity to fix the syntax for this piece of code, the full code related to issues.db. It is ill-formed and needs to be fixed. @MDTsai if you want you can do it here. Or I will do it after in #1294 (as you wish).